web/src/routes/admin/submissions/[submissionId]/+page.svelte

```diff
@@ -16,12 +16,82 @@
 	export let form: Actions;
 
 	let confirmModal: ConfirmModal;
-	let gradingMessage: HTMLTextAreaElement;
 
 	$: if (form && form.success) {
 		goto('/admin/reviews');
 	}
 
+	type JudgeResponseCategory = {
+		title: string;
+		message: string;
+		correct: boolean | null;
+		description?: string;
+	};
+
+	let chosenJudgeResponse: JudgeResponseCategory | null = null;
+
+	const customResponseCategory: JudgeResponseCategory = {
+		title: 'Custom',
+		message: '',
+		correct: null
+	};
+
+	const judgeResponseCategories: JudgeResponseCategory[] = [
+		{
+			title: 'Output Incorrect',
```

Comment:
I know the title/message properties all match, but I think maybe the message text should be more descriptive and thus wanted to keep these separate for now.